.github/workflows/tests.yml

```diff
@@ -0,0 +1,108 @@
+name: Test Suite
+on: [push, pull_request]
+
+jobs:
+    test:
+        runs-on: ${{matrix.operating-system}}
+        strategy:
+            matrix:
+                operating-system: [ ubuntu-latest ]
+                php: [ "5.6", "7.0", "7.1", "7.2", "7.3", "7.4" ]
+        name: PHP ${{matrix.php }} Unit Test
+        steps:
+            - uses: actions/checkout@master
```

Comment:
I might be tempted to use this at a specific tag, I think we use `v2` for our Node.js repos (just so you don't get any surprise regressions).